.gitignore

```diff
@@ -2,3 +2,5 @@ build
 
 site/
 
+# editor specific
+.vscode
```

Comment:
You should add this to your local `.gitignore`. I think that we don't need it on remote repository.